template/greeter/Dockerfile

```diff
@@ -2,10 +2,12 @@
 
 FROM --platform=${BUILDPLATFORM:-<%- platformName %>} node:18-alpine AS dev-deps
 
+RUN corepack enable pnpm
+RUN pnpm config set store-dir /workdir/.pnpm-store
+
 WORKDIR /workdir
 
 COPY package.json yarn.lock ./
 
-RUN \
-  --mount=type=secret,id=npm,dst=/root/.npmrc \
-  yarn install --frozen-lockfile --ignore-optional --non-interactive
+RUN --mount=type=secret,id=npm,dst=/root/.npmrc pnpm fetch
+RUN pnpm install --offline --recursive
```

Comment:
@samchungy Did you end up running `pnpm install` in your Dockerfile?

Trying to figure out the best way forward, especially where we [build a second Dockerfile for Gantry interop](https://github.com/seek-oss/skuba/blob/master/template/koa-rest-api/Dockerfile) 🥲

Reply:
Yeah I opted for simplicity:

https://github.com/SEEK-Jobs/backoffice-budgets-api/pull/224/files#diff-02e5f267970af3f4a889c28bba25f5a39b956dcd12a541a62340bbd06e0cafecL11